runtime/caml/domain.h

```diff
@@ -85,7 +87,7 @@ CAMLextern void (*caml_domain_initialize_hook)(void);
 CAMLextern void (*caml_domain_stop_hook)(void);
 CAMLextern void (*caml_domain_external_interrupt_hook)(void);
 
-CAMLextern void caml_init_domains(uintnat minor_heap_wsz);
+CAMLextern void caml_init_domains(uintnat minor_heap_wsz, uintnat max_domains);
```

Comment:
Nit: I would maybe expect the other parameter order, as `max_domains` is the most important parameter of the two.

Reply:
Done in https://github.com/ocaml/ocaml/pull/13272/commits/263be361baad53f6ef36f8774ac8b5d9b1572525.